man/man1/exiv2.1

```diff
@@ -682,7 +682,7 @@ the YYYY:MM:DD HH:MM:SS standard.
 .sp 1
 .ne 40
 .SH CONFIGURATION FILE
-Exiv2 can read an optional configuration file ~/.exiv2 on Unix systems and %USERPROFILE%\\exiv2.ini on Windows (using a Visual Studio build).  Cygwin and MinGW/msys2 follow the unix convention and use ~/.exiv2  You can determine the location of the configuration file with the command:
```

Comment:
2 things I'm not clear on:
1. Not sure why this specifically mentions "Visual Studio build"
2. it says MinGW and the others use a unix convention, but that's not what was implemented in the code... I've no clue what would make more sense for these platforms... 

@kevinbackhouse @alexvanderberkel @piponazo any suggestions?

Reply:
I don't know the history of this. I would remove "(using a Visual Studio build)" as it doesn't seem to add anything.

Reply:
I went ahead and changed the man to reflect what is actually done in the code.   

@kevinbackhouse  that of course dismissed your review again :sweat_smile: So could you just approve again, and I'll go ahead and finally merge this :)